doc/release/README.prereqs

```diff
@@ -4,7 +4,7 @@ Chapel Prerequisites and Assumptions
 
 If you have a standard UNIX environment, a C/C++ compiler, some basic
 scripting languages, a GNU-compatible make, and awk installed you should
```

Comment:
Thanks.  I removed one reference to awk and overlooked this one.  Will fix.

Mike

On Mar 24, 2015, at 3:21 PM, Brad Chamberlain <notifications@github.com<mailto:notifications@github.com>> wrote:


In doc/release/README.prereqs<https://github.com/chapel-lang/chapel/pull/1705#discussion_r27079539>:

> @@ -4,7 +4,7 @@ Chapel Prerequisites and Assumptions
>
>  If you have a standard UNIX environment, a C/C++ compiler, some basic
>  scripting languages, a GNU-compatible make, and awk installed you should


Hi @noakesmichael<https://github.com/noakesmichael> : Did I misunderstand our lunchtime conversation? It looks like 'awk' is still named in this line and the three at the bottom of the file? Maybe there's another pending pull request I haven't seen yet...?

—
Reply to this email directly or view it on GitHub<https://github.com/chapel-lang/chapel/pull/1705/files#r27079539>.